dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/util/PeriodOffsetUtils.java

```diff
@@ -194,9 +196,6 @@ public static DataQueryParams removeOffsetPeriodsIfNotNeeded( DataQueryParams pa
         BiFunction<Integer, Integer, Integer> replaceIndexIfMissing = ( Integer index,
```

Comment:
Seems `replaceIndexIfMissing` can be removed.

Reply:
Done, thanks!